os/drivers/Kconfig

```diff
@@ -282,6 +282,41 @@ if DRIVERS_VIDEO
 source drivers/video/Kconfig
 endif # DRIVERS_VIDEO
 
+menuconfig WIFI_CSI
+	bool "CSI Support"
+	default n
+	---help---
+		Enable support for WIFI CSI driver.
+
+if WIFI_CSI
+
+config WIFICSI_CUSTOM_DEV_PATH
+	string "Use custom device path"
+	default "/dev/wificsi"
+
+if WIFICSI_CUSTOM_DEV_PATH
+
+config WIFICSI_DEV_ROOT
+	bool "Place wificsi devices in /dev"
+	default n
+	---help---
+		This option causes all device entries to appear in /dev with all the
+		other device entries.  This option generates the smallest code and
+		RAM footprint.
+
+if !WIFICSI_DEV_ROOT
+
+config WIFICSI_DEV_PATH
```

Comment:
What is difference between `WIFICSI_CUSTOM_DEV_PATH` and this?

Reply:
Obsolete, it has been removed now.